packages/vuetify/src/components/VChip/VChip.tsx

```diff
@@ -0,0 +1,208 @@
+// Styles
+import './VChip.sass'
+
+// Composables
+import { makeTagProps } from '@/composables/tag'
+import { useColor } from '@/composables/color'
+
+// Directives
+import { Ripple, RippleDirectiveBinding } from '@/directives/ripple'
+
+// Components
+import VIcon from '@/components/VIcon/VIcon'
+// import { VExpandXTransition } from '../transitions'
+
+// Utilities
+import { computed, defineComponent, withDirectives } from 'vue'
+import makeProps from '@/util/makeProps'
+import { useDirective } from '@/util/useDirective'
+import { makeSizeProps, useSize } from '@/composables/size'
+
+export default defineComponent({
+  name: 'VChip',
+
+  props: makeProps({
+    active: {
+      type: Boolean,
+      default: true,
+    },
+    activeClass: {
+      type: String,
+      default: '',
+    } as any,
```

Comment:
Why as any?

Reply:
This was leftover from a PropValidation type that was there.